Dockerfile

```diff
@@ -101,7 +101,9 @@ RUN addgroup -S gerbera 2>/dev/null && \
     adduser -S -D -H -h /var/run/gerbera -s /sbin/nologin -G gerbera -g gerbera gerbera 2>/dev/null && \
     addgroup gerbera video && \
     mkdir /var/run/gerbera/ && chmod 2775 /var/run/gerbera/ && \
-    mkdir /content && chmod 777 /content
+    mkdir /content && chmod 777 /content && ln -s /content /mnt/content && \
```

Comment:
From your description, I'd expect `mkdir /mnt/content` here.

Reply:
Sorry, I might not have explained it well.

The idea is to allow users to keep their configuration. They can still mount a volume into `/content`, and Gerbera will read media from there. That is because `/mnt/content` is a symlink to `/content`.

If a new user comes along, however, she can mount a volume into `/mnt/content`, which replaces the symlink by the volume.

In both cases, Gerbera will read from `/mnt/content`. The difference is that this "directory" is either a _symlink_ to a volume, or a volume. I have tested it and it works like a charm.

Did I make more sense this time?

Reply:
I have updated `Dockerfile.debug` as well - it had escaped my notice.

Reply:
> Sorry, I might not have explained it well.
> 
> The idea is to allow users to keep their configuration. They can still mount a volume into `/content`, and Gerbera will read media from there. That is because `/mnt/content` is a symlink to `/content`.
> 
> If a new user comes along, however, she can mount a volume into `/mnt/content`, which replaces the symlink by the volume.
> 
> In both cases, Gerbera will read from `/mnt/content`. The difference is that this "directory" is either a _symlink_ to a volume, or a volume. I have tested it and it works like a charm.
> 
> Did I make more sense this time?

But there is no autoscan entry for /mnt/content

Reply:
> I have updated `Dockerfile.debug` as well - it had escaped my notice.
> 
> _Other than the previous changes I have not tested this change as `scripts/install-googletest.sh` quit with a server error._

That is some caching issue, goes away if you run the build again.

Reply:
> But there is no autoscan entry for /mnt/content

You are right, I must have missed it when I partitioned my changes into small logical commits. Weird that my tests succeeded, though.

This issue is now fixed, and I have taken the opportunity to improve the readability of the `sed` invocations.